indico/modules/users/models/users.py

```diff
@@ -12,15 +12,9 @@
 from operator import attrgetter
 from uuid import uuid4
 
-from flask import flash, g, has_request_context, session
 from flask_multipass import IdentityRetrievalFailed
-from sqlalchemy.dialects.postgresql import JSONB, UUID
-from sqlalchemy.event import listens_for
-from sqlalchemy.ext.associationproxy import association_proxy
-from sqlalchemy.ext.hybrid import hybrid_property
-from sqlalchemy.orm import object_session
-from werkzeug.utils import cached_property
```

Comment:
Please undo this

Reply:
you mean... to keep the imports, right?

Reply:
keeping them as they are, since the previous sorting was correct (not sure why your isort tries to move them like this)

Reply:
I changed the imports to this:
```
import itertools
from enum import Enum
from operator import attrgetter
from uuid import uuid4

from flask import flash, g, has_request_context, session
from flask_multipass import IdentityRetrievalFailed
from sqlalchemy.dialects.postgresql import JSONB, UUID
from sqlalchemy.event import listens_for
from sqlalchemy.ext.associationproxy import association_proxy
from sqlalchemy.ext.hybrid import hybrid_property
from sqlalchemy.orm import object_session
from werkzeug.utils import cached_property

from indico.core import signals
from indico.core.auth import multipass
from indico.core.db import db
from indico.core.db.sqlalchemy import PyIntEnum
from indico.core.db.sqlalchemy.custom.unaccent import define_unaccented_lowercase_index
from indico.core.db.sqlalchemy.principals import PrincipalType
from indico.core.db.sqlalchemy.util.models import get_default_values
from indico.modules.users.models.affiliations import UserAffiliation
from indico.modules.users.models.emails import UserEmail
from indico.modules.users.models.favorites import favorite_category_table, favorite_user_table
from indico.util.i18n import _
from indico.util.locators import locator_property
from indico.util.string import format_full_name, format_repr, return_ascii
from indico.util.struct.enum import RichIntEnum
from indico.web.flask.util import url_for
```

... but now I cannot commit: the pre-commit checks are blocking me.
I already tried:
git diff --staged --name-only --diff-filter d indico/indico/modules/users/models/users.py | xargs isort

...but nothing changed.
Suggestions?

Reply:
Something must be wrong with your local setup if isort tries to mess up your imports. Maybe the virtualenv is not active? Anyway, `git commit -n` to skip checks is your friend (at least as a workaround).

Reply:
Thank you, it worked